product-service/serverless.ts

```diff
@@ -63,6 +130,39 @@ const serverlessConfiguration: Serverless = {
             path: '/products',
             cors: true
           }
+        },
+        {
+          sqs: {
+            batchSize: 1,
```

Comment:
Batch size should be 5, this is a requirement stated in the task description.

Reply:
Yes, I updated MR, now it has 5 batchSize. I set it as 1 for having only one event.Records size